app/main.py

```diff
@@ -2,0 +1,51 @@
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
Similarly, the `skills` attribute in `FrontendDeveloper` is a class variable. This will be shared across all instances of `FrontendDeveloper`. Consider moving it to the constructor to make it an instance variable.